packages/regl-worldview/src/commands/Command.js

```diff
@@ -84,9 +84,12 @@ export default class Command<T> extends React.Component<Props<T>> {
 
   _getHitmapPropFromHitmapId(objectId: number) {
     const { hitmapProps = [] } = this.props;
+
     return hitmapProps.find((hitmapProp) => {
-      // TODO handle objects with 'colors' property
-      if (hitmapProp.color) {
+      if (hitmapProp.points) {
+        // just return the original hitmapProp for instanced rendering
+        return true;
```

Comment:
if there are multiple components with instanced rendering, doesn't this just always fire off or am I misunderstanding something? It seems like we'll have to calculate the range/set of ids that each `hitmapProp` can produce, and if the `objectId` is within that range/set, we can return `true`.

Reply:
Good point! totally missed it.